src/main/java/nl/goodbytes/xmpp/xep0363/SecureUUID.java

```diff
@@ -65,5 +68,11 @@ public int hashCode() {
 		}
 		return Objects.hash(this.id);
 	}
+	
+	public static void main(String...args) throws Exception {
```

Comment:
Let's not have `public static void main()` in this code :) You might want to add it as a unit test instead?

Reply:
ok. it was trivial so I removed it. fad8c4fd8ee402245b29122752b31e556c5ad96a